### PR TITLE
Add Error Popover Page

### PR DESCRIPTION
**Pull request by evancrow**
Fixed: #2914

Here's how it looks:

Now we have another issue of making the errors make sense...

### CODE REVIEW

**Comment by evancrow** [comment]
Added an optional description field:

**Comment by evancrow** [comment]
Discussed with @yozuysal, going to make this a Toast instead. This requires the user to tap something and read an error they won't understand or be helpful.

**Comment by EdwardJTL** [comment]
could you update the screenshots on the PR when you get a chance?

**Comment by evancrow** [comment]
Here's a recording @EdwardJTL:

https://user-images.githubusercontent.com/37356984/155815175-d00b8877-1b8d-4a1f-a2a2-c11fd62de2ad.mp4

File: Client.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -7352,7 +7352,6 @@
 			buildSettings = {
 				CODE_SIGN_ENTITLEMENTS = Client/Entitlements/Neeva.entitlements;
 				DEFINES_MODULE = YES;
-				"EXCLUDED_ARCHS[sdk=iphonesimulator*]" = "";
```
Comment (EdwardJTL) [inline Client.xcodeproj/project.pbxproj:7355]: where did these changes come from? we should move them to a separate PR if they need to be checked in.

Comment (evancrow) [reply Client.xcodeproj/project.pbxproj:7355]: I was testing running from an M- Mac, must have made it in unintentionall. Thanks for catching.

**Review by EdwardJTL — APPROVED** [review]
